bal_addresses/subgraph.py

```diff
@@ -31,7 +37,7 @@ def get_subgraph_url(self, subgraph="core") -> str:
             if chain == "zkevm":
                 return "https://subgraph.satsuma-prod.com/ab0804deff79/1xhub-ltd/aura-finance-zkevm/api"
             elif chain in ["avalanche"]:  # list of chains without an aura subgraph
-                return None
+                return ""
```

Comment:
Why this change?  Should think about a consistent no result in the library if possible.

Reply:
can we still return none and catch that in the gen_subgraph script?